### ISSUE TITLE
WINDOWS OS: NVM not working once Multiple VS Code integrated terminals open

### ISSUE THREAD

**Issue by IGilman** [microsoft/vscode#181341]
I recently install NVM for Windows following these instructions: https://learn.microsoft.com/en-us/windows/dev-environment/javascript/nodejs-on-windows

Overall, things work when using an external GitBash terminal and/or using GitBash in the VS Code integrated terminal. BUT once I open a 2nd VS Code terminal, the 2nd project no longer let's me run Node or Angular commands via the integrated terminal. Even if I close all the VS Code instances and re-open one, then every time I try to run a command I am getting "command not found errors". 

I never have these issues when using an external bash terminal. 

Also, when I restart my computer, the one project that wasn't working works fine. It's always the second project integrated terminal that doesn't work.

Here is one example of it happening: Picture of terminals (2 integrated, 1 external)
![image](https://user-images.githubusercontent.com/10121038/235715294-bb9c1fa4-2c43-46ed-a966-8582982b496f.png)

I've tried restarting my computer multiple times. I've tried to figure out if it was a certain project that wasn't working. I've searched for a little, but most solutions are Mac focused

Also, I've checked that my Bash path's are the same in my terminal v. my Vs Code terminal settings:

![image](https://user-images.githubusercontent.com/10121038/235715611-c0722303-8c61-4336-aa6b-7fa55777e4a0.png)

**Comment by meganrogge** [comment]
this means those programs are not on your `$PATH`.  what happens if you comment out all of your settings?

**Comment by IGilman** [comment]
Commands are not found 

![image](https://user-images.githubusercontent.com/10121038/235928982-05250d7f-d653-4aba-a6f2-88a27cb7f6da.png)

**Comment by meganrogge** [comment]
do you have any interesting startup scripts for either bash or powershell?

**Comment by IGilman** [comment]
@meganrogge I don't think so. So weird thing. After I commented out the settings and un-commented them, plus a computer restart, the issue is no longer happening. I am unsure why tho.

**Comment by meganrogge** [comment]
this is probably related to `nvm` aliases not being defined in these terminals. You should check that configuration

**Comment by IGilman** [comment]
@meganrogge so I've run into the issue again. I still don't understand why it worked for a little and now it's stopped. 

Here is what I am seeing. 

When it works in VS Code (and in my external terminal, which always works), here is the $PATH:

![NodeNvm_ExternalTerminal](https://user-images.githubusercontent.com/10121038/236026627-575e37e2-c4a8-4103-84e7-443dbbcdbc51.png)

![NodeNvm_VsCode](https://user-images.githubusercontent.com/10121038/236026645-9a0355e0-de82-4f99-8dda-4101e7150630.png)


When it stops working in VS Code here is what the $PATH looks like: 
![NodeNvm_VsCode_Broken](https://user-images.githubusercontent.com/10121038/236026662-32b34288-2ff8-4aaf-a658-7e3dcad7de7b.png)

I don't know how to check the aliases in my terminal, but maybe this is the same issue? I just don't really understand why it works sometimes, especially after a restart and only in one VS Code instance, and doesn't work other times, especially after I open a second VS Code Instance.

**Comment by IGilman** [comment]
Ok I think I solved the problem. For some reason, once I opened up another VS Code instance, my $PATH variables/directory locations were no longer found (based on my previous screen shots). Here is what I mean:

On the left is my normal user account and the right is my admin account that we use to download programs etc at work. Because I had to download everything with my admin account, it probably wrote all my $PATH variables to that account settings, and not my normal user account.

So I manually added the directory locations in my Path variables on the right side and I think the issue is resolved.

![image](https://user-images.githubusercontent.com/10121038/236214817-50d80395-0763-496c-835b-53051a70fb07.png)
